backend/DAO/matchDAO.js

```diff
@@ -53,5 +53,25 @@ module.exports = {
                     throw new SuperError(ERR_INTERNAL_SERVER_ERROR, 'There has been a problem retrieving your matches. Please try again')
                 }
             })
+    },
+
+    deleteMatch: (studentId, placementId) => {
+        return database('student_has_placement as shp')
+            .where('student_id', studentId)
+            .andWhere('placement_id', placementId)
+            .del();
+    },
+
+    getMatchesByPlacementId: (placementId) => {
+        return database('student_has_placement as shp')
+            .select('shp.student_id as studentId')
```

Comment:
Here is the place where we have to add to the query also the rest of the info that we need from the DB